<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ticket</class>
 <widget class="QWidget" name="ticket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#lbl_start,#lbl_end{color:rgb(85,255,255)}
QLabel#lbl_before,#lbl_flghtid{color:#0985d9}
QLabel#lbl_1,#lbl_2,#lbl_info{color:#f6bd40}
QPushButton#btn_cancel{background:#ff3333}
QLabel#lbl_date{color:white;font:18px}
QLabel#lbl_during{color: transparent}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>机票</string>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>681</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QLabel" name="lbl_flightid">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_time_start">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;06:45&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_time_end">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;09:00&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_start">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>白云</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_end">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>虹桥</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_during">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2小时15分</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_arrow">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res.qrc">:/res/arrow_right.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancel">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>30</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>退票</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_before">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>空客A01</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_date">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>发车日期：</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_info">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>161</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>机票仅当日当次有效,</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>191</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>请注意：距离航班开始</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_time">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;还有不到24小时&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>100</y>
       <width>251</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>请妥善安排您的时间！</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
